pkg/generate/git/repository.go

```diff
@@ -364,7 +364,7 @@ func command(name, dir string, env []string, args ...string) (stdout, stderr str
 func timedCommand(timeout time.Duration, name, dir string, env []string, args ...string) (stdout, stderr string, err error) {
 	var stdoutBuffer, stderrBuffer bytes.Buffer
 
-	glog.V(4).Infof("Executing %s %s %s", strings.Join(env, " "), name, strings.Join(args, " "))
+	glog.V(4).Infof("Executing %s %s", name, strings.Join(args, " "))
```

Comment:
anyone who can view the build logs can generally access any information that might appear here.